src/index.html

```diff
@@ -1,16 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="#"
+      method="post"
+    >
+      <label class="label1">
+        <img
+          class="image1"
+          src="./images/Search.svg"
+          alt="Search"
+        >
+
+        <input
+          data-qa="keypress"
+          class="text1"
+          type="text"
+          name="img"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
```

Comment:
no spaces between child and parent

Reply:
may you please tell me which line you are talking about ?